hub/api/tensor.py

```diff
@@ -56,7 +55,7 @@ def __setitem__(self, item: Union[int, slice], value: np.ndarray):
                 "Assignment to Tensor slices not currently supported!"
             )
         else:
-            write_array(
+            add_samples_to_tensor(
```

Comment:
Same here. This should overwrite and not append

Reply:
ok i will handle this

Reply:
im adding it so that it throws an error, we should update the API in a separate PR to handle overwriting